packages/babel-compiler/babel-compiler.js

```diff
@@ -195,7 +195,7 @@ BCp._inferFromBabelRc = function (inputFile, babelOptions, cacheDeps) {
       try {
         this._babelrcCache[babelrcPath] = {
           controlFilePath: babelrcPath,
-          controlFileData: JSON.parse(
+          controlFileData: JSON5.parse(
```

Comment:
Maybe update the error message below to `.babelrc is not a valid JSON5 file`? :slightly_smiling_face:

Reply:
kk changed it